homeassistant/components/motioneye/__init__.py

```diff
@@ -440,6 +478,68 @@ async def handle_webhook(
     return None
 
 
+def _get_media_event_data(
+    hass: HomeAssistant,
+    device: dr.DeviceEntry,
+    event_file_path: str,
+    event_file_type: int,
+) -> dict[str, str]:
+    config_entry_id = next(iter(device.config_entries), None)
+    client = hass.data[DOMAIN].get(config_entry_id, {}).get(CONF_CLIENT)
+    coordinator = hass.data[DOMAIN].get(config_entry_id, {}).get(CONF_COORDINATOR)
```

Comment:
Assuming we have a guard that returns as soon as the config entry is missing

```suggestion
    data = hass.data[DOMAIN][config_entry_id]
    client = data[CONF_CLIENT]
    coordinator = data[CONF_COORDINATOR]
```